scripts/spectra/spectra.py

```diff
@@ -106,6 +110,41 @@ def plot_spectra(
     plt.savefig(filename)
 
 
+def plot_spectra_nm(
+    filename, spectra, E=np.linspace(3.5, 7.0, 1000), delta=0.05,
+):
```

Comment:
Add function documentation as per comment above.

Reply:
Documentation for these functions is in /docs/user_scripts/SCRIPTNAME.md

Reply:
I mean the function documentation with docstrings, the annotations of the values that are passed to the functions, and a description of the parameters and return values. Following this convention for the package will make it possible for people to understand what's happening with the code. Additionally, rather than having to write additional documentation (as you so nicely did in `/docs/user_scrips/SCRIPTNAME.md`) we should just include that information in the code itself, and then we can auto-generate the documentation from the code. 

So I'd take the writeup in `convert-geomdat.md` and add it as the docstring to the top of the script itself, and then let the `mkdocs` library do the document generation itself. I'll do this for you on convert_to_geomdat.py as an example.

Reply:
Actually, upon further inspection, his script doesn't work at all and hard-codes a ton of files that obvisouly won't exist for any user. I the code needs to be removed or refactored to actually work. Thoughts?

Reply:
Yeah it's not really ideal, it's just a useful utility if you happen to have like... 200 folders of single point energies and want to make an absorption spectrum. It used to be in 'util' but I moved it to scripts kinda for this reason; I think a lot of people in the group use it. But it's independent of any kind of trajectory object; it's specifically for a workflow prior to AIMS.